shared/entities.js

```diff
@@ -0,0 +1,5758 @@
+
+/* eslint-disable quote-props */
```

Comment:
Let's analyse the perf cost to injecting this into the extension (along with the tldts)? This is why is why it was avoided before.